### PR TITLE
Adding support for a keyboard called ortho4exent 

### PR DESCRIPTION
**Pull request by tszaboo**
## Description

Adding support for a keyboard called ortho4exent 
QMK MSYS compiled the hex, and I was able to upload it to the board.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/tszaboo/ortho4exent/readme.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# tszaboo
+
+Ortholinear PCB that is a drop-in replacement for Exent keyboard.
+Layout is inspired by excellent Boardwalk PCB, but extended with an extra column to accommodate the 65% Exent form factor.
+
+Keyboard Maintainer: [tszaboo](https://github.com/tszaboo/)
+Keyboard Product owner: [Riba]
+Hardware Supported: Ortho4exent
+Hardware Availability: [Geekhack Group Buy](https://geekhack.org/index.php?topic=110120.0)
```
Comment (zvecr) [inline keyboards/tszaboo/ortho4exent/readme.md:9]:
```suggestion
* Keyboard Maintainer: [tszaboo](https://github.com/tszaboo/)
* Keyboard Product owner: [Riba]
* Hardware Supported: Ortho4exent
* Hardware Availability: [Geekhack Group Buy](https://geekhack.org/index.php?topic=110120.0)
```

File: keyboards/tszaboo/ortho4exent/rules.mk [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+MCU = atmega32u4
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
```
Comment (zvecr) [inline keyboards/tszaboo/ortho4exent/rules.mk:11]:
```suggestion
```

File: keyboards/tszaboo/ortho4exent/rules.mk [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+LTO_ENABLE = yes
+VIA_ENABLE = yes
```
Comment (zvecr) [inline keyboards/tszaboo/ortho4exent/rules.mk:35]:
```suggestion
# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/tszaboo/ortho4exent/info.json [hunk 4]
```diff
@@ -0,0 +1,87 @@
+{
+    "keyboard_name": "Ortho4exent",
+    "url": "https://github.com/tszaboo/Ortho4Exent",
+    "maintainer": "tszaboo",
+    "width": 16,
+    "height": 5,
+    "layouts": {
+      "LAYOUT": {
+        "key_count": 71,
```
Comment (zvecr) [inline keyboards/tszaboo/ortho4exent/info.json:9]:
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/tszaboo/ortho4exent/config.h [hunk 5]
```diff
@@ -0,0 +1,218 @@
+/*
+Copyright 2019 mechmerlin
```
[the remark this answers is not in the document]
Comment (tszaboo) [reply keyboards/tszaboo/ortho4exent/config.h:2]: Ok ,changed

**Comment by tszaboo** [comment]
After merging your changes, I was able to build the firmware with QMK MSYS

File: keyboards/tszaboo/ortho4exent/config.h [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
```
Comment (zvecr) [inline keyboards/tszaboo/ortho4exent/config.h:2]:
```suggestion
Copyright 2021 tszaboo
```

File: keyboards/handwired/onekey/blackpill_f411/config.h [hunk 7]
```diff
@@ -1,4 +1,4 @@
-/* Copyright 2019
+/* Copyright 2021 tszaboo
```
Comment (zvecr) [inline keyboards/handwired/onekey/blackpill_f411/config.h:1]:
```suggestion
/* Copyright 2019
```

File: keyboards/tszaboo/ortho4exent/rules.mk [hunk 8]
```diff
@@ -0,0 +1,25 @@
+# MCU name
+MCU = atmega32u4
+BOOTLOADER = caterina
+
```
Comment (zvecr) [inline keyboards/tszaboo/ortho4exent/rules.mk:4]:
```suggestion
# Bootloader selection
BOOTLOADER = caterina
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/tszaboo/ortho4exent/config.h (hunk 6 above)
Comment (tszaboo) [reply keyboards/tszaboo/ortho4exent/config.h]: resolved

File: keyboards/tszaboo/ortho4exent/rules.mk [hunk 9]
```diff
@@ -0,0 +1,25 @@
+# MCU name
+MCU = atmega32u4
+# Bootloader selection
```
Comment (fauxpark) [inline keyboards/tszaboo/ortho4exent/rules.mk:3]:
```suggestion

# Bootloader selection
```

File: keyboards/tszaboo/ortho4exent/rules.mk [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+LTO_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/tszaboo/ortho4exent/rules.mk:25]:
```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = yes        # Console for debug
COMMAND_ENABLE = yes        # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
LTO_ENABLE = yes
```

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Comment by tszaboo** [comment]
Thank you guys, I appreciate the help!

**Pull request merged**